packages/client/hmi-client/src/components/tabs/TabContainer.vue

```diff
@@ -22,6 +23,8 @@ const props = defineProps<{
 	activeTabIndex: number;
 }>();
 
+console.log(props.tabs); // Doesn't like the props.tabs.props here - I get extraneous non-props attribute warning from this
```

Comment:
I might be misunderstanding but maybe it's because `tabs` is an array? So it would be
```
props.tabs[0].props
```

But could also be that the `props` word is reserved in Vue, we could try renaming the property and see if it still yells.

Reply:
We will let @echl fix this when is tabs PR is done